test/test_gem_compatibility.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+# JRuby does not support forking, and it doesn't seem worth the effort to make it work.
+return unless Process.respond_to?(:fork)
```

Comment:
Active Support has some [workarounds](https://github.com/rails/rails/blob/608c1bfe17a2626304b6aae96a3824517fc07bb2/activesupport/lib/active_support/testing/isolation.rb) for this we could use, but it doesn't seem worth it for a code path that was previously untested, and is unlikely to behave differently on JRuby.